lib/app_web/live/item_live/index.html.heex

```diff
@@ -0,0 +1,45 @@
+<h1>Listing Items</h1>
+
+<%= if @live_action in [:new, :edit] do %>
+  <.modal return_to={Routes.item_index_path(@socket, :index)}>
```

Comment:
@SimonLab do we _need_ to use a `<.modal>` for creating `new` `items`? 🤷‍♂️

Reply:
The modal is created automatically by Phoenix when running the `phx.gen.live` command at https://github.com/dwyl/learn-alpine.js/blob/phoenix-example-%234/drag-and-drop.md#create-items

I thought about updating the way Phoenix creates the items however I didn't want to make the tutorial longer and preferred to focus on the drag and drop feature part itself.

Reply:
@SimonLab thanks for clarifying that this was generated by `phx.gen.live`. 👌 
It's _definitely_ concerning [to me] that someone in the Phoenix team 🤷‍♂️ 
thought it was "OK" to include **modals** in the default UX for creating new content. 🤦‍♂️ 
Modals are not even remotely "OK"; they're a UX anti-pattern 😢 
and a major red flag of lazy design. 🔴 

For the purposes of this example/spike, 🧑‍💻 
the objective is to demo drag-and-drop 🐉 
and you have achieved that. ✅ 

In terms of our scarce/valuable time we don't _need_ to spend any more on this. 💸 
I've made a note to write about it: https://github.com/dwyl/product-ux-research/issues/38 📝 
We will _completely_ avoid them in our **`App`**. 🙅